rpc/bridge.go

```diff
@@ -70,6 +72,65 @@ func NewBridgeEndpoints(
 	}
 }
 
+// TokenMappingsResult contains the token mappings and the total count of token mappings
+type TokenMappingsResult struct {
+	TokenMappings []*bridgesync.TokenMapping `json:"tokenMappings"`
+	Count         int                        `json:"count"`
+}
+
+// GetTokenMappings returns the token mappings for the given network
+func (b *BridgeEndpoints) GetTokenMappings(networkID uint32, pageNumber, pageSize *uint32) (interface{}, rpc.Error) {
+	b.logger.Debugf("GetTokenMappings invoked (network id=%d, page number=%v, page size=%v)",
+		networkID, pageNumber, pageSize)
+
+	ctx, cancel := context.WithTimeout(context.Background(), b.readTimeout)
+	defer cancel()
+
+	c, merr := b.meter.Int64Counter("get_token_mappings")
```

Comment:
@Stefan-Ethernal whats the usecase for this counter? do we track anything with this?

Reply:
I have just followed what we have in the already existing [functions](https://github.com/agglayer/aggkit/blob/509353b885176e54a0e2f0a8b95e29a53a5646cc/rpc/bridge.go#L80) since we have exposed metrics for the bridge service, which can be useful in the future, once it is being run in the production (to have some metrics that indicate the API invocations rate).